### PR TITLE
SOMMELIER-14

### PR DESCRIPTION
**Pull request by jnannni**

### CODE REVIEW

File: App/components/Navbar/navItem.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React, { ReactNode } from 'react'
+import * as Label from '../Typography'
+import * as Styled from './navbar.styled'
+import { FONT_SIZE_KEYS } from '../../theme/fontSizes'
+import { COLOR_KEYS } from '../../theme/colors'
+
+export type Props = {
+  onPress: () => void
+  text?: string
+  selected: boolean
```
Comment (StanislavMyakishev) [inline App/components/Navbar/navItem.tsx]: Please consider removing selected prop as it not used now

**Review by StanislavMyakishev — APPROVED** [review]

File: svg.d.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+declare module '*.svg' {
+  import React from 'react'
+  import { SvgProps } from 'react-native-svg'
+  const content: React.FC<SvgProps>
+  export default content
+}
```
Comment (Mixxx-a) [inline svg.d.ts:6]: I've moved this file into App directory, so you don't really need this file

**Review by Mixxx-a — APPROVED** [review]
Delete svg.d.ts because it is duplication of code

**Review by Mixxx-a — COMMENTED** [review]
And don't forget to add export of component to root's `index.ts`

**Pull request merged**